### ISSUE TITLE
Это файл с логами?

### ISSUE THREAD

**Issue by Aleksey2093** [RadmirSad/GDPI_forward#2]
### Describe your feature / Опишите ваше предложение

Это файл с логами?

https://github.com/RadmirSad/GDPI_forward/blob/main/src/out_ip_list.txt

**Comment by RadmirSad** [comment]
Да, логи для дебага помещал в отдельный файл, т.к. логи сыпались очень быстро, и нормально анализировать их было проблематично
Только не уверен, что это актуальные логи. Файл с ними вообще не должен был сюда выкатываться
